src/components/TokenEnterAmount.test.tsx

```diff
@@ -1,55 +1,32 @@
-import { fireEvent, render } from '@testing-library/react-native'
+import { act, fireEvent, render, renderHook } from '@testing-library/react-native'
+import BigNumber from 'bignumber.js'
 import React from 'react'
 import { getNumberFormatSettings } from 'react-native-localize'
 import { Provider } from 'react-redux'
-import { AmountEnteredIn } from 'src/send/types'
-import { StoredTokenBalance, TokenBalance } from 'src/tokens/slice'
-import { NetworkId } from 'src/transactions/types'
+import { LocalCurrencySymbol } from 'src/localCurrency/consts'
+import { useTokenInfo } from 'src/tokens/hooks'
+import { TokenBalance } from 'src/tokens/slice'
+import { convertLocalToTokenAmount, convertTokenToLocalAmount } from 'src/tokens/utils'
 import { createMockStore } from 'test/utils'
-import {
-  mockCeloTokenBalance,
-  mockCeloTokenId,
-  mockEthTokenId,
-  mockTestTokenTokenId,
-  mockTokenBalances,
-} from 'test/values'
-import TokenEnterAmount, { APPROX_SYMBOL } from './TokenEnterAmount'
+import { mockCeloTokenBalance, mockCusdTokenBalance, mockUSDCTokenId } from 'test/values'
+import TokenEnterAmount, {
+  APPROX_SYMBOL,
+  formatNumber,
+  getDisplayLocalAmount,
+  getDisplayTokenAmount,
+  useEnterAmount,
+} from './TokenEnterAmount'
 
 jest.mock('react-native-localize')
+jest.mock('src/analytics/AppAnalytics')
+jest.mock('src/redux/hooks', () => ({ useSelector: jest.fn() }))
+jest.mock('src/tokens/hooks', () => ({ useTokenInfo: jest.fn() }))
```

Comment:
now that i write this, i don't think you need to use `useTokenInfo` at all in the hook since it receives the whole `token` object as a prop....

Reply:
@kathaypacific good point! I was copying it from the usage of `useTokenToLocalAmount/useLocalToTokenAmount` back at the very beginning of the refactor and haven't revisited it. Will adjust!

Reply:
@kathaypacific you're right! I was using it at the start when I removed the usages of `useTokenToLocalAmount/useLocalToTokenAmount` hooks and haven't revisited it since.